src/App.js

```diff
@@ -0,0 +1,35 @@
+import { Routes, Route } from 'react-router-dom';
+import { AnimatePresence } from 'framer-motion';
+import styled from 'styled-components';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+
+import { fetchStocks } from './redux/stocks/thunks/stock';
+import Details from './components/pages/Details';
+import Home from './components/pages/Home';
+
+const AppContainer = styled.div`
+  @media (min-width: 768px) {
+    padding: 0 250px;
+  }
+`;
+const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchStocks());
+    // eslint-disable-next-line
```

Comment:
- [ ] Kindly remove the ESLint disable here. ESLint is there to help you write better code and disabling it defies its purpose. Please fix the linter check on this part 👍